client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts

```diff
@@ -146,6 +144,11 @@ const useInvestigationTable = (parameters: useInvestigationTableParameters): use
             })
     }
 
+    const canChangeStatusNewToInProcess = (investigationStatus: Number, investigationInvestigator? : string) => {
+        const investigatorTocheck = investigationInvestigator || currInvestigatorId;
```

Comment:
Is there any way an investigation won’t have creator? And if There is, curr investigation won’t be useful anyway because in the landing page we don’t have a current investigation

Reply:
Good point! I removed current investigation's creator. 
Either way, creator is not  nullable